pkg/wekafs/apiclient/apiclient.go

```
@@ -94,11 +97,13 @@ func (cm *WekaCompatibilityMap) fillIn(versionStr string) {
 	f, err := version.NewVersion(MinimumSupportedWekaVersions.FilesystemAsVolume)
 	q, err := version.NewVersion(MinimumSupportedWekaVersions.QuotaDirectoryAsVolume)
 	n, err := version.NewVersion(MinimumSupportedWekaVersions.QuotaOnNonEmptyDirs)
+	a, err := version.NewVersion(MinimumSupportedWekaVersions.AuthenticatedMount)
```

better names please

---

also, errors are not handled